Unresolved external symbol on compiling lib on v1.42 on Windows

compile with msvc v16.8:
command:

```bash
make -f makefile.msvc
```

log:

```

link -nologo -MAP -debug -incremental:no -opt:ref,icf -MANIFEST  -dll -out:MSVC_obj/nghttp2.dll -implib:MSVC_obj/nghttp2.lib MSVC_obj/r_nghttp2_pq.obj MSVC_obj/r_nghttp2_map.obj MSVC_obj/r_nghttp2_queue.obj MSVC_obj/r_nghttp2_frame.obj MSVC_obj/r_nghttp2_buf.obj MSVC_obj/r_nghttp2_stream.obj MSVC_obj/r_nghttp2_outbound_item.obj MSVC_obj/r_nghttp2_session.obj MSVC_obj/r_nghttp2_submit.obj MSVC_obj/r_nghttp2_helper.obj MSVC_obj/r_nghttp2_npn.obj MSVC_obj/r_nghttp2_hd.obj MSVC_obj/r_nghttp2_hd_huffman.obj MSVC_obj/r_nghttp2_hd_huffman_data.obj MSVC_obj/r_nghttp2_version.obj MSVC_obj/r_nghttp2_priority_spec.obj MSVC_obj/r_nghttp2_option.obj MSVC_obj/r_nghttp2_callbacks.obj MSVC_obj/r_nghttp2_mem.obj MSVC_obj/r_nghttp2_http.obj MSVC_obj/r_nghttp2_rcbuf.obj -PDB:MSVC_obj/nghttp2.pdb MSVC_obj/r_nghttp2.res
   Creating library MSVC_obj/nghttp2.lib and object MSVC_obj/nghttp2.exp
r_nghttp2_map.obj : error LNK2019: unresolved external symbol nghttp2_ksl_init referenced in function map_insert
r_nghttp2_map.obj : error LNK2019: unresolved external symbol nghttp2_ksl_free referenced in function nghttp2_map_free
r_nghttp2_map.obj : error LNK2019: unresolved external symbol nghttp2_ksl_insert referenced in function map_insert
r_nghttp2_map.obj : error LNK2019: unresolved external symbol nghttp2_ksl_remove referenced in function nghttp2_map_remove
r_nghttp2_map.obj : error LNK2019: unresolved external symbol nghttp2_ksl_lower_bound referenced in function nghttp2_map_find
r_nghttp2_map.obj : error LNK2019: unresolved external symbol nghttp2_ksl_begin referenced in function nghttp2_map_each_free
r_nghttp2_map.obj : error LNK2019: unresolved external symbol nghttp2_ksl_len referenced in function nghttp2_map_each_free
r_nghttp2_map.obj : error LNK2019: unresolved external symbol nghttp2_ksl_it_get referenced in function nghttp2_map_each_free
MSVC_obj\nghttp2.dll : fatal error LNK1120: 8 unresolved externals
make: *** [MSVC_obj/nghttp2.dll] Error 1120

```

--- a/lib/Makefile.msvc
+++ b/lib/Makefile.msvc
@@ -91,7 +91,8 @@ NGHTTP2_SRC := nghttp2_pq.c \
   nghttp2_callbacks.c \
   nghttp2_mem.c \
   nghttp2_http.c \
-  nghttp2_rcbuf.c
+  nghttp2_rcbuf.c \
+  nghttp2_ksl.c
 
 NGHTTP2_OBJ_R := $(addprefix $(OBJ_DIR)/r_, $(notdir $(NGHTTP2_SRC:.c=.obj)))
 NGHTTP2_OBJ_D := $(addprefix $(OBJ_DIR)/d_, $(notdir $(NGHTTP2_SRC:.c=.obj)))

